feature-libs/organization/administration/core/services/b2b-user-creation-notifier.service.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { B2BUser } from '@spartacus/core';
+import { Observable, Subject } from 'rxjs';
+import { LoadStatus } from '../model';
+
+interface UserCreationNotificationData {
```

Comment:
`UserCreationNotifierData` sounds better to me. But it's up to you. :)

Reply:
Couldn't be also this part of the model file?

Reply:
Changed